src/doc/rustc/src/platform-support.md

```diff
@@ -146,6 +146,7 @@ target | std | notes
 `armv7a-none-eabi` | * | Bare ARMv7-A
 `armv7r-none-eabi` | * | Bare ARMv7-R
 `armv7r-none-eabihf` | * | Bare ARMv7-R, hardfloat
+`armv8r-none-eabihf` | * | Bare ARMv8-R, hardfloat
```

Comment:
I think this target with its own caveats and all would benefit from a dedicated target page describing major points (such as around the floating point support.)

Yes, the `armv7r` targets don’t have one, but it wouldn’t be a terrible time to start – it’ll also be a great place to note down as to who the maintainer(s) of the target are.

Reply:
Thanks, I'll take a stab at this.

Reply:
Speaking of... who are the maintainers of the various arm-none targets?

Reply:
Unfortunately we don't have great records here as many of these targets predate this process. For now, I would just document that you are the maintainer of this specific target.